nss-capricious-finches/nutshell
# Rb login button

# Description of Proposed Changes
logInLogic.js
Takes the user log in and filters the database for the specific user, if the user doesn't exist it says we haven't been introduced.

## Steps to Test
Outline the steps to test

```
git fetch --all
git checkout RB-loginButton
```

run json server and grunt
with console open check to make sure that when you log in with random name it says "sorry we haven't been introduced"
Or if you log in using a fake real user JSmith11
it should welcome you and clear the dom
**this will need to be updated with the module that creates the app

# Impacted Areas in Application
List general components of the application that this PR will affect:
logInLogic.js
APIManager.js
storage.js
login.js (i had to add id's)

# Mentions @username
@paulzimmclay @JacobAndrewSmith92 

# Definition of Done
#### The project must be fully documented. This includes the following:
  1) Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
  2) Every class must be documented with purpose, author, and methods.
  3) Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
  4) The project must be able to run fully, and without errors, on each teammate's system.
  5) Fulfills every requirement. Every line of code has been peer reviewed.
  6) For projects that require unit testing, core functionality must be identified and have at least one test for each.
